### ISSUE TITLE
login error after update to rc4

### ISSUE THREAD

**Issue by j00ls** [mailpile/Mailpile#2201]
After updating to mailpile rc4 I can't log in anymore

web front end:
I can start mailpile fine but the Passphrase only gives me a "Oops wrong password.." message

command line:
Failed to load language C
Bad passphrase for None
login error: Invalid password, please try again

What happened and can I fix it? Thanks!

**Comment by j00ls** [comment]
I did a general update with the mailpile rc4 update and that must somehoe be related... 
My first thoughts was there might be problems with pgp or the key - but then it shouldn't just be saying "invalid password" no?

**Comment by BjarniRunar** [comment]
Were you definitely able to log in before the update, or had you just left Mailpile running forever?

**Comment by j00ls** [comment]
I was even able to log in once after the update. Then I restarted the server and since then it has been giving me the "invalid password". I even logged in once via the console and once via the web interface after I updated mailpile without problems.
So I assumed everything went well - and restarted the server.

**Comment by BjarniRunar** [comment]
Note that updating the .deb or the git tree doesn't automatically restart Mailpile.

Since you didn't say you restarted Mailpile I'm going to assume you didn't. So things were probably immediately broken on update, but how/why is hard for me to say. You've said rc4, but nothing about how you installed (deb package? github? 3rd party package), so I dunno.

Sometimes people have reported this problem if GnuPG isn't installed (or gets uninstalled).

**Comment by BjarniRunar** [comment]
Closing this, without further details there's nothing I can do here. Thank you for filing this, please feel free to re-open or file another issue if you feel I was hasty.

**Comment by ManfredWisniewski** [comment]
Sorry for taking so long... I started a new job and things have been pretty buisy. I do have some new leads:

1.) I deleted the mailpile user directory and created a new one: no problems
2.) I replaced the mailpile.key of the new user directory with the old one and I can log in with my old password - so gnpg is not the GnuPG is not the problem and I also do not have the wrong password.

I then tried replacing the files in the new directory with my backup files from the old mailpile user-directory. The strange thing: I did this as root via sudo and I get the following error:

Cannot open source file "..." Permission denied (13)

Does this help in any way? Thanks!
